cmd/nvkind/cluster_create.go

```diff
@@ -135,6 +174,19 @@ func runClusterCreate(c *cli.Context, f *ClusterCreateFlags) error {
 		}
 	}
 
+	// Display endpoints against which kubectl is configured, for debug.
+	args := []string{"kubectl", "cluster-info", "--context", "kind-" + cluster.Name}
+	if err := runChildProcess(args, ""); err != nil {
+		return fmt.Errorf("running process: %w", err)
+	}
```

Comment:
It feels like this (and the function below) should be wrapped in a function inside the `cluster` package and just called from this file rather than defining these operations here.

Reply:
ack, doing that now

Reply:
great suggestion, learned some things while doing so